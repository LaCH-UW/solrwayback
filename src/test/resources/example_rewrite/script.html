<html>
<head>
    <title>Some page</title>
    <link rel="stylesheet" type="text/css" href="styles/index.css" />
</head>
<h1>A very simple page</h1>

<p>An <a href="http://example.com">absolute link</a> leading to example.com</p>
<p>A <a href="subfolder/">relative link</a> leading to <tt>subfolder</tt>subfolder</tt></p>
<p>A <a href="/otherpage.html">root-based link</a> leading to a parallel page <tt>otherpage.html</tt>.</p>

<p>An absolute referenced image: <img src="http://example.com/image.jpg"/></p>
<p>A relatively referenced image: <img src="subfolder/image.jpg"/></p>
<p>A root referenced image: <img src="/image.jpg"/></p>

<p>An image with absolute srcset: <img src="http://example.com/foo.jpg" srcset="http://example.com/foo.jpg 1x, http://example.com/foo2.jpg 2x"/></p>
<p>An image with relative srcset: <img src="foo.jpg" srcset="foo.jpg 1x, foo2.jpg 2x"/></p>
<p>An image with root srcset: <img src="/foo.jpg" srcset="/foo.jpg 1x, /foo2.jpg 2x"/></p>

</html>
