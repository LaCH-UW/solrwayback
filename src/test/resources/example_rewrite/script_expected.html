<html lang="en">
 <head>
  <title>Some page with JavaScript</title>
 </head>
 <body>
  <h1>To support a not-to-be-named large social media site and video playback</h1>
  <script type="application/json" id="foo1">{"uri":"http:\/\/localhost:0000\/solrwayback\/services\/downloadRaw?source_file_path=somesourcefile&amp;offset=0"}</script>
  <!-- Note that forward slashes are escaped.This is legal and an easy way to avoid accidentally writing </ -->
  <script>
 someFunction([[{
 "profile_picture":{"uri":"http:\/\/localhost:0000\/solrwayback\/services\/downloadRaw?source_file_path=somesourcefile&amp;offset=1","width":40,"height":40,"scale":1},
 "playable_url_dash":"http:\/\/localhost:0000\/solrwayback\/services\/downloadRaw?source_file_path=somesourcefile&amp;offset=2",
 "video": "\u003CBaseURL&gt;http:\/\/localhost:0000\/solrwayback\/services\/downloadRaw?source_file_path=somesourcefile&amp;offset=3\u003C\/BaseURL&gt;",
 "playable_url":"http:\/\/localhost:0000\/solrwayback\/services\/downloadRaw?source_file_path=somesourcefile&amp;offset=4",
 "playable_url_quality_hd":"http:\/\/localhost:0000\/solrwayback\/services\/downloadRaw?source_file_path=somesourcefile&amp;offset=5",
 "url":"http:\/\/localhost:0000\/solrwayback\/services\/downloadRaw?source_file_path=somesourcefile&amp;offset=6"
 }]]);
</script>
 </body>
</html>